### PR TITLE
Init the module

### PR DESCRIPTION
**Pull request by WuxuanChen**
Init the module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,167 @@
+Terraform module to implements Multi-Account Single Sign-On by Cloud SSO.
+
 # terraform-alicloud-landing-zone-cloudsso
-tbd
+
+English | [简体中文](./README-CN.md)
```
Comment (shanye997) [inline README.md]: https://github.com/alibabacloud-automation/terraform-alicloud-landing-zone-cloudsso/blob/main/README-CN.md

File: README.md [hunk 2]
```diff
@@ -1,2 +1,167 @@
...
+
+When enterprises adopt a multi-account cloud architecture, configuring personnel identities and permission policies for each account becomes very cumbersome and time-consuming. This module leverages the product capabilities of cloud SSO to provide a centralized way to manage identities and permissions across multiple accounts, simplifying the complexity of identity configuration for enterprise IT administrators.
+
+![Structure](./pictures/structure.png)
```
Comment (shanye997) [inline README.md]: https://raw.githubusercontent.com/alibabacloud-automation/terraform-alicloud-demilitarized-zone-design/main/pictures/structure.png

File: README.md [hunk 3]
```diff
@@ -1,2 +1,167 @@
...
+      access_configurations = ["Admin"],
+      accounts = [data.alicloud_resource_manager_accounts.default.accounts[0].account_id]
+    }
+  ]
+}
+```
+
+## Examples
+
+- [Basic Example](./examples/basic/README.md)
```
Comment (shanye997) [inline README.md]: https://github.com/alibabacloud-automation/terraform-alicloud-landing-zone-cloudsso/tree/main/examples/basic

File: README.md [hunk 4]
```diff
@@ -1,2 +1,167 @@
...
+- [Complete Example](./examples/complete/README.md)
```
Comment (shanye997) [inline README.md]: https://github.com/alibabacloud-automation/terraform-alicloud-landing-zone-cloudsso/tree/main/examples/complete

File: README-CN.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
+Terraform module to implements Multi-Account Single Sign-On by Cloud SSO.
+
+# terraform-alicloud-landing-zone-cloudsso
+
+[English](./README.md) | 简体中文
```
Comment (shanye997) [inline README-CN.md]: https://github.com/alibabacloud-automation/terraform-alicloud-landing-zone-cloudsso/blob/main/README.md

File: README-CN.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+当企业采用多账号的云上架构时，配置各个账号的人员身份和权限策略将变得非常繁琐耗时。此模块基于云SSO的产品能力，提供一种集中化管理企业多账号的身份权限的方式，简化了企业IT管理人员在身份配置上的复杂度。
+
+![架构图](./pictures/structure-CN.png)
```
Comment (shanye997) [inline README-CN.md]: https://raw.githubusercontent.com/alibabacloud-automation/terraform-alicloud-demilitarized-zone-design/main/pictures/structure-CN.png

File: README-CN.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+      principal_type = "Group",
+      access_configurations = ["Admin"],
+      accounts = [data.alicloud_resource_manager_accounts.default.accounts[0].account_id]
+    }
+  ]
+}
+```
+
+## 示例
+- [基础用法](./examples/basic/README.md)
```
Comment (shanye997) [inline README-CN.md]: 同英文文档

File: modules/cloud_sso_access_assignment/variables.tf
Comment (shanye997) [inline modules/cloud_sso_access_assignment/variables.tf:1]: 变量名加引号。
变量都需要添加 default 值，没有就写 null

File: main.tf
Comment (shanye997) [inline main.tf:1]: 所有代码都需要格式化，执行  terraform fmt -recursive

File: README-CN.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+
+![架构图](https://raw.githubusercontent.com/alibabacloud-automation/terraform-alicloud-landing-zone-cloudsso/main/pictures/structure-CN.png)
+
+## 前提条件
+
+开通阿里云云SSO服务
+
+## 用法
+
+通过Terraform管理阿里云云SSO内的用户、用户组、访问配置以及多账号授权。
```
Comment (wibud) [inline README-CN.md:17]: 需要补充下，对于已经通过SCIM同步等方式，创建了User和Group，怎么去跳过Module里的User和Group创建步骤

File: examples/basic/main.tf [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+  groups = [
+    {
+      group_name = "tf-example-group1"
+    }
+  ]
+
+  access_configurations = [
+    {
+      access_configuration_name = "Admin"
```
Comment (wibud) [inline examples/basic/main.tf:25]: 示例中，建议将权限的配置也放进去。比如配置一个系统权限

File: main.tf [hunk 10]
```diff
@@ -0,0 +1,43 @@
+data "alicloud_cloud_sso_directories" "default" {}
```
Comment (wibud) [inline main.tf:1]: 建议将cloudsso的开通加进去，同时要判断下是否已经开通，避免重复开通报错

**Pull request merged**